examples/spine/index.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+ <!doctype html>
```

Comment:
they were added to trip CI to run the test suite over these files :) (I will drop the commit before I manually merge)

Our CI now is really smart and will only run the integration tests over the examples that have been changed, however when you make changes to the test files... no tests run so you need to manually  touch some files in projects (it is a bit of a hack... however I have not had the time / motivation to make it less of a hack ((any help welcome here)))